massa-execution-worker/src/execution.rs

```diff
@@ -577,6 +578,7 @@ impl ExecutionState {
                 coins: *coins,
                 owned_addresses: vec![sender_addr],
             }];
+            context.datastore = (*datastore).clone();
```

Comment:
If I'm not mistaken, just "datastore.clone()" should deref implicitly

Reply:
done